CONTRIBUTING.md

```diff
@@ -35,9 +36,32 @@ times in N different commits. If there was some accidental reformatting or white
 changes during the course of your commits, please rebase them away before submitting
 the PR.
 
+### CHANGELOG using Beachball
+
+[Beachball](https://microsoft.github.io/beachball/) is a semantic version bumper that also has an automated tool to ask devs to log changes in a simple CLI manner.
+
+The TeamsJS Client SDK v2 repo contains a changelog (file is named `CHANGELOG.md`) for substantial changes in the `<root>/packages/teams-js` directory. If you make any changes to the `<root>/packages/teams-js` directory (you can see the exception files and directories in `<root>/beachball.config.js`), you must run `yarn generate-change-files` or `yarn beachball` from the monorepo root to generate change files.
```

Comment:
I think it's clearer to say: "The TeamsJS Client SDK v2 `2.0-preview` branch contains..." since we're in the same repo.

Reply:
Done, will be included in the next commit(s)! (I ended up tweaking it so there is no v2 in the sentence)

Reply:
Updated